persistence/nhibernate/index.md

```diff
@@ -15,9 +15,16 @@ related:
 Uses the [NHibernate ORM](https://nhibernate.info/) for persistence.
 
 
-## Supported persistence types
-
-partial: supported-persistence
+## Persistence at a glance
+
+|Feature                    |   |
+|:---                       |---
+|Supported Storage Types    |Sagas, Outbox, Subscriptions, Timeouts
+|Unsupported Storage Types  |None
+|Transactions               |SQL transactions
+|Saga Concurrency Control   |Optimistic concurrency for correctness + pessimistic concurrency for performance
+|Scripted Deployment        |Not supported
```

Comment:
Script generator is part of the package and we have doco showing how to generate scripts as part of unit tests

Reply:
@SzymonPobiega in NHibernate? Then what is this? https://docs.particular.net/persistence/nhibernate/#generating-scripts-for-deployment

Reply:
😱  we seem to not have documentation for it :( https://github.com/Particular/NServiceBus.NHibernate/blob/master/src/NServiceBus.NHibernate.Tests/GenerateScriptsTest.cs

Reply:
I'll see if I can sneak that in after this PR during our existing docs enhancement release window.